app/src/main/resources/application.properties

```diff
@@ -67,3 +81,16 @@ quarkus.index-dependency.jaxrs.artifact-id=jboss-jaxrs-api_2.1_spec
 %prod.registry.kafka.snapshot-producer.acks=all
 %prod.registry.kafka.snapshot.requests=1000
 %prod.registry.kafka.snapshot-consumer.bootstrap.servers=localhost:9092
+# Streams -- when used
+%prod.registry.streams.topology.bootstrap.servers=localhost:9092
+%prod.registry.streams.topology.application.id=apicurio-registry
+%prod.registry.streams.topology.application.server=localhost:9000
+%prod.registry.streams.topology.num.stream.threads=2
+%prod.registry.streams.topology.num.standby.replicas=1
+%prod.registry.streams.topology.processing.guarantee=exactly_once
+%prod.registry.streams.topology.replication.factor=1
+%prod.registry.streams.storage-producer.bootstrap.servers=localhost:9092
+%prod.registry.streams.storage-producer.enable.idempotence=true
+#%prod.registry.streams.storage-producer.max.in.flight.requests.per.connection=5
+%prod.registry.streams.storage-producer.retries=3
+%prod.registry.streams.storage-producer.acks=all
```

Comment:
Should any of the prod configuration values for Kafka be set at runtime via environment variables?

Reply:
I image at the very least the various `servers` properties would need to be.  @alesj ?

Reply:
Yeah, bootstrap.servers is where Kafka is running, so this one is definitely env var.
The others are local to consumer/client config, and we should find good defaults (with a way to override them ...).

Reply:
>> Yeah, bootstrap.servers is where Kafka is running, so this one is definitely env var.
I'll fix this as part of my PR ... and check if there are others ...